server/services/store/sqlstore/data_migrations.go

```diff
@@ -547,3 +549,91 @@ func (s *SQLStore) getBoardUserTeams(tx sq.BaseRunner, board *model.Board) (map[
 
 	return userTeams, nil
 }
+
+func (s *SQLStore) runDeletedMembershipBoardsMigration() error {
+	if !s.isPlugin {
+		return nil
+	}
+
+	setting, err := s.GetSystemSetting(DeletedMembershipBoardsMigrationKey)
+	if err != nil {
+		return fmt.Errorf("cannot get deleted membership boards migration state: %w", err)
+	}
+
+	// If the migration is already completed, do not run it again.
+	if hasAlreadyRun, _ := strconv.ParseBool(setting); hasAlreadyRun {
+		return nil
+	}
+
+	boards, err := s.getDeletedMembershipBoards(s.db)
+	if err != nil {
+		return err
+	}
+
+	s.logger.Debug("Migrating boards with owner not anymore on their team", mlog.Int("count", len(boards)))
+
+	tx, err := s.db.BeginTx(context.Background(), nil)
```

Comment:
Perhaps don't create a transaction if len(boards) == 0?

Reply:
I guess this doesn't matter if we're not going to run this migration every time.

Reply:
Anyhow, fixed in https://github.com/mattermost/focalboard/pull/3635/commits/d23e6e41b09cf82775dbc646b66f908312875677 :)